<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Climbing area access test app...</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonTicket">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ticket [F1]</string>
    </property>
    <property name="shortcut">
     <string>F1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEntry">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Entry [F2]</string>
    </property>
    <property name="shortcut">
     <string>F2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInfo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>211</y>
      <width>116</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Info [F3]</string>
    </property>
    <property name="shortcut">
     <string>F3</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>216</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="layoutAlarms">
       <item>
        <widget class="QLabel" name="labelAlarms">
         <property name="text">
          <string>Alarms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxAlarms">
         <property name="toolTip">
          <string>Anzahl der Alarme</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutEntries">
       <item>
        <widget class="QLabel" name="labelEntries">
         <property name="text">
          <string>Entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxEntries">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutUnused">
       <item>
        <widget class="QLabel" name="labelUnused">
         <property name="text">
          <string>Unused</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxUnused">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelStateDoor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Door State</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelStateScanner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scanner State</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1244445</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="placeholderText">
     <string>member number</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
